crates/volta-core/src/tool/package/install.rs

```diff
@@ -27,7 +27,7 @@ use volta_fail::{throw, Fallible, ResultExt};
 //       installed packages.
 /// Configuration information about an installed package.
 ///
-/// This information will be stored in ~/.volta/tools/user/packages/<package>.json.
+/// This information will be stored in ~/.volta/tools/default/packages/<package>.json.
```

Comment:
We shouldn't change the filesystem layout for this change. We want to update the internal code to say default, but changing the filesystem is a bigger change that likely isn't worth it. So we should leave this comment the same.

Reply:
Ah yes I was struggling with the file system name before, should ask you before the PR :)